### PR TITLE
Added custom metrics to Widget Builder

### PR DESCRIPTION
**Pull request by ale-cota**
## DESCRIBE YOUR PR
Added custom metrics to Dashboards / Widget Builder

## IS YOUR CHANGE URGENT?  

Help us prioritize incoming PRs by letting us know when the change needs to go live.
- [ ] Urgent deadline (GA date, etc.): 
- [ ] Other deadline: 
- [x] None: Not urgent, can wait up to 1 week+

## PRE-MERGE CHECKLIST

*Make sure you've checked the following before merging your changes:*

- [x] Checked Vercel preview for correctness, including links
- [x] PR was reviewed and approved by any necessary SMEs (subject matter experts)
- [x] PR was reviewed and approved by a member of the [Sentry docs team](https://github.com/orgs/getsentry/teams/docs)

### CODE REVIEW

File: docs/product/dashboards/widget-builder/index.mdx [hunk 1]
```diff
@@ -74,6 +74,18 @@ Choosing "Releases (sessions, crash rates)" allows you to query properties speci
...
+
+Choosing "Custom Metrics" allows you to query your custom metrics data in the same way you would in the [Metrics Explorer Page](/product/metrics/metrics-explorer-page/). Some example widgets might include:
+
+- Tracking your number of user signups
+- Tracking API request payloads
+- Processing time for a task
+
+Please note that the widget builder UI is slightly different than for other datasets, to allow for an optimal experience when querying metrics.
+
+![Metrics widget builder.](./img/metrics-widget-builder.png)
```
Comment (ale-cota) [inline docs/product/dashboards/widget-builder/index.mdx:87]:
```suggestion
![Custom metrics widget builder.](./img/widget-builder-custom-metrics.png)
```

File: docs/product/dashboards/widget-builder/index.mdx [hunk 2]
```diff
@@ -74,6 +74,18 @@ Choosing "Releases (sessions, crash rates)" allows you to query properties speci
...
 
+### Custom Metrics
+
+Choosing "Custom Metrics" allows you to query your custom metrics data in the same way you would in the [Metrics Explorer Page](/product/metrics/metrics-explorer-page/). Some example widgets might include:
+
+- Tracking your number of user signups
+- Tracking API request payloads
+- Processing time for a task
+
+Please note that the widget builder UI is slightly different than for other datasets, to allow for an optimal experience when querying metrics.
```
Comment (ale-cota) [inline docs/product/dashboards/widget-builder/index.mdx:85]:
```suggestion
Please note that the widget builder UI is slightly different than for other datasets in order to optimize the experience for querying metrics.
```

**Review by vivianyentran — APPROVED** [review]
Reworded this a bit, but thanks for adding!

**Review by shellmayr — APPROVED** [review]

File: docs/product/dashboards/widget-builder/index.mdx [hunk 3]
```diff
@@ -74,6 +74,18 @@ Choosing "Releases (sessions, crash rates)" allows you to query properties speci
 - Crash rates for your latest release
 - Session count across releases
 
+### Custom Metrics
+
+Choosing "Custom Metrics" allows you to query your custom metrics data in the same way you would in the [Metrics Explorer Page](/product/metrics/metrics-explorer-page/). Some example widgets might include:
```
[the remark this answers is not in the document]
Comment (ale-cota) [reply docs/product/dashboards/widget-builder/index.mdx:79]: @vivianyentran that's actually not quite so 😅 you can still also plot OOTB metrics like transaction.duration and others.

**Pull request merged**